test/cql-pytest/test_keyspace.py

```diff
@@ -97,15 +97,22 @@ def test_drop_keyspace_nonexistent(cql):
         cql.execute('DROP KEYSPACE nonexistent_keyspace')
 
 # Test trying to ALTER a keyspace.
-def test_alter_keyspace(cql, this_dc):
+def test_alter_keyspace(cql, this_dc, has_tablets):
     with new_test_keyspace(cql, "WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', '" + this_dc + "' : 1 }") as keyspace:
-        cql.execute(f"ALTER KEYSPACE {keyspace} WITH REPLICATION = {{ 'class' : 'NetworkTopologyStrategy', '{this_dc}' : 3 }} AND DURABLE_WRITES = false")
+        if has_tablets:
+            repl_opts = ""
+        else:
+            repl_opts = f" REPLICATION = {{ 'class' : 'NetworkTopologyStrategy', '{this_dc}' : 3 }} AND "
+        cql.execute(f"ALTER KEYSPACE {keyspace} WITH " + repl_opts + " DURABLE_WRITES = false")
```

Comment:
nit: if you're using the `f""` string notation, why use `+` to concatenate multiple strings?
`f"ALTER KEYSPACE {keyspace} WITH {repl_opts} DURABLE_WRITES = false"` would be simpler